lambdas/restore-vcs/src/main/java/uk/gov/di/ipv/core/restorevcs/RestoreVcsHandler.java

```diff
@@ -178,8 +180,10 @@ private void sendVcRestoredAuditEvent(String userId, VcStoreItem vcStoreItem)
                         jwtClaimsSet.getIssuer(),
```

Comment:
I think this is another case where we can use the `getExtensionsForAudit()` from the `library.auditing.helpers`